sdks/python/apache_beam/pipeline.py

```diff
@@ -520,10 +520,26 @@ def run(self, test_runner_api='AUTO'):
             self._options,
             allow_proto_holders=True).run(False)
 
+      if (self._options.view_as(TypeOptions).runtime_type_check and
+          self._options.view_as(TypeOptions).performance_runtime_type_check):
+        raise RuntimeError(
```

Comment:
I don't see a use case. The existing `runtime_type_check` flag does everything that `performance_runtime_type_check` does but more, since it runtime type-checks everything instead of sampling. If a user turns on both, they lose performance in 2 ways with no added benefit coming from the `performance_runtime_type_check`. What do you think?

Reply:
I think eventually we'll want to consolidate the two. I suppose we can prohibit setting both if we want.

Reply:
Sounds good